examples/oft-solana/README.md

```diff
@@ -100,14 +100,18 @@ anchor keys sync
 
 :warning: `--force` flag overwrites the existing keys with the ones you generate.
 
-Run `anchor keys list` to view the generated programIds (public keys). The output should look something like this:
+Run 
+```
+anchor keys list
+```
+to view the generated programIds (public keys). The output should look something like this:
 
 ```
 endpoint: <ENDPOINT_PROGRAM_ID>
 oft: <OFT_PROGRAM_ID>
 ```
 
-Copy the OFT's programId and go into [lib.rs](./programs/oft/src/lib.rs). Note the following snippet:
+Copy the OFT's programId and go into [lib.rs](./programs/oft/src/lib.rs) (located inside this project under oft-solana/programs/oft/src/lib.rs). Note the following snippet:
```

Comment:
@lzdane I don't think there's a need for the note in parentheses, the link already links to the  right file. Devs can click on it. It's normal assumption that a filename with a link in a README links to a file in the same repo.